config.yaml

```diff
@@ -0,0 +1,88 @@
+# Main gamestate and step to load
```

Comment:
This should not be in here. It also shouldn't be possible for this to be in here.